### PR TITLE
Update documentation to include .Net Core

### PR DESCRIPTION
**Pull request by ipalvr**
Customer asked if we support .Net Core and sent me a link to this doc.  They sent screen shots of .Net Core being scanned in their repo.  I also looked in Slack and saw others asking if this was supported, therefore, submitting this request to have the documentation updated.

### CODE REVIEW

File: compute/admin_guide/continuous_integration/code_repo_scanning.adoc [hunk 1]
```diff
@@ -9,6 +9,7 @@ The runtimes supported are:
 * Node.js
 * Python
 * Ruby
+* C# (.NET Core 2.1, .NET Core 3.1)
```
Comment (Pubs-MV) [inline compute/admin_guide/continuous_integration/code_repo_scanning.adoc:12]: @tallieber , please confirm. 

Related but not the same issue.
This is also contradictory to what we see specifically for serverless runtimes in the UI per https://spring.paloaltonetworks.com/twistlock/twistlock/issues/33209
vs
the runtimes displayed in the Open API Spec file for downloading serverless bundles: https://prisma.pan.dev/api/cloud/cwpp/defenders#operation/post-defenders-serverless-bundle
image

Would be great to get the correct list for what's supported.

Comment (Pubs-MV) [reply compute/admin_guide/continuous_integration/code_repo_scanning.adoc:12]: cc: @rcaballeromx , @ssugandh

**Comment by rcaballeromx** [comment]
@Pubs-MV I've assigned this issue to myself and @ssugandh for us to more easily find it among our tickets.

File: compute/admin_guide/continuous_integration/code_repo_scanning.adoc (hunk 1 above)
Comment (ipalvr) [reply compute/admin_guide/continuous_integration/code_repo_scanning.adoc:12]: Thanks all, if we support  .Net Framework (4.x) based libraries/executables, and .Net Core (5.x, 6.0) based libraries/executables should we add those as well?

Comment (Pubs-MV) [reply compute/admin_guide/continuous_integration/code_repo_scanning.adoc:12]: Hi @ipalvr , I believe @avish9 had told me that `only 3.1 and 6.0 are supported (4.0/5.0 arent supported).` It was in a review comment here: https://docs.google.com/document/d/1AhlagzchynFUOUe3D-v_zBiwiQ84oQ1z/edit

Please confirm @avish9

**Comment by ssugandh** [comment]
Here's the ticket for the engg. team for verification.
https://spring.paloaltonetworks.com/twistlock/twistlock/issues/39747
I haven't assigned to anyone but it would be good to get a list of all the supported values for the body parameter `runtime`

File: compute/admin_guide/continuous_integration/code_repo_scanning.adoc (hunk 1 above)
Comment (avish9) [reply compute/admin_guide/continuous_integration/code_repo_scanning.adoc:12]: The versions I mentioned relate to Runtime Protection - Serverless Defender for Azure.

**Comment by iansk** [comment]
@rcaballeromx @ssugandh What's the status of this PR?

**Comment by ssugandh** [comment]
@iansk: I'm not sure about the Admin Guide but the API reference has this information here: https://prisma.pan.dev/api/cloud/cwpp/defenders#operation/post-defenders-serverless-bundle

The cURL example seems to be published in a wrong way. I'll rectify that in another PR. 
cURL example corrected here: https://github.com/PaloAltoNetworks/prisma-cloud-docs/pull/503

**Comment by Pubs-MV** [comment]
old backlog. closing PR as there have been many updates since this PR was created.

**Pull request closed without merging**